proto/store/setting.proto

```diff
@@ -33,6 +33,27 @@ message WorkspaceProfileSetting {
 
   // The duration for refresh token.
   google.protobuf.Duration refresh_token_duration = 6;
+
+  // The setting of custom announcement
+  Announcement announcement = 7;
+}
+
+message Announcement {
+  // We support three levels of AlertLevel: INFO, WARNING, and ERROR.
+  enum AlertLevel {
+    ALERTLEVEL_INFO = 0;
```

Comment:
No need to have ALERTLEVEL prefix here, otherwise the generated type is: Announcement_AlertLevel.ALERTLEVEL_INFO

Reply:
According https://protobuf.dev/programming-guides/proto3/#enum, it would be better to keep ALERTLEVEL prefix.